src/components/Activity/ActivityButton.js

```diff
@@ -0,0 +1,132 @@
+import React, { useCallback, useRef, useState } from 'react'
+import { Spring, animated } from 'react-spring/renderprops'
+import {
+  ButtonIcon,
+  IconAlert,
+  Popover,
+  GU,
+  Tag,
+  springs,
+  useTheme,
+} from '@aragon/ui'
+import ActivityList from './ActivityList'
+import { useActivity } from './ActivityProvider'
+
+const ActivityButton = React.memo(function ActivityButton() {
+  const theme = useTheme()
+  const [opened, setOpened] = useState(false)
+  const { markActivitiesRead, unreadCount } = useActivity()
+  const containerRef = useRef()
+
+  const handleToggle = useCallback(
+    () =>
+      setOpened(opened => {
+        if (opened) {
+          markActivitiesRead()
+        }
+        return !opened
+      }),
+    [markActivitiesRead]
+  )
+  const handleClose = useCallback(() => {
+    markActivitiesRead()
+    setOpened(false)
+  }, [markActivitiesRead])
+
```

Comment:
What do you think if implementing this same logic ? 
 https://github.com/aragon/court-dashboard/blob/d8e5e46e8a8983eb58ad056455d53a195a3926b4/src/components/Account/Account.js#L138-L144

But with `activities` as dependency, so when clearing activities we don't lose focus.

Reply:
Yes good idea! Done in https://github.com/aragon/court-dashboard/pull/100/commits/a32c00c905034dfb7661d67f23bf8f2207ea70ec